packages/orama/src/components/sorter.ts

```diff
@@ -54,7 +61,8 @@ function innerCreate(schema: Schema, sortableDeniedProperties: string[], prefix:
         sorter.sortableProperties.push(path)
         sorter.sortablePropertiesWithTypes[path] = type
         sorter.sorts[path] = {
-          docs: {},
+          docs: new Map(),
+          orderedDocsToRemove: new Map(),
```

Comment:
We can use a map here as long as we don't need to serialize this property to JSON. As far as I understand, we can leave this property as Map.

Reply:
Yep, it is not serialize with JSON.stringify 

```
% node
Welcome to Node.js v20.2.0.
Type ".help" for more information.
> JSON.stringify({ a: new Map() })
'{"a":{}}'
> a = new Map()
Map(0) {}
> a.set('a', 0)
Map(1) { 'a' => 0 }
> JSON.stringify({ a })
'{"a":{}}'
> a
Map(1) { 'a' => 0 }
> 
```

But probably we can use a custom toJSON function to avoid this issue.

Reply:
Yep, we don't need to serialize this property.